python/examples/jupyter/Katana Tutorial.ipynb

```diff
@@ -0,0 +1,419 @@
+{
+ "cells": [
```

Comment:
For later, but I found this Python package which runs notebooks and checks their output with an existing notebook. I haven't used it myself, but maybe it will be helpful.

https://pypi.org/project/pytest-notebook/

Reply:
Oh nice! I'll put this on my radar.

Reply:
I really love the idea of being able to test out tutorial notebooks. This makes me must more excited than is rational. :-)